server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerUpdateMetrics.java

```diff
@@ -68,10 +69,14 @@ public void addMutationArraySize(long value) {
 
   @Override
   public void registerMetrics(MeterRegistry registry) {
-    permissionErrorsCounter = registry.counter(METRICS_UPDATE_ERRORS, "type", "permission");
-    unknownTabletErrorsCounter = registry.counter(METRICS_UPDATE_ERRORS, "type", "unknown.tablet");
-    constraintViolationsCounter =
-        registry.counter(METRICS_UPDATE_ERRORS, "type", "constraint.violation");
+    FunctionCounter.builder("tabletServerUpdateErrors", permissionErrorsCount, AtomicLong::get)
```

Comment:
I think the names still need to be in the standard form, `accumulo.tserver.updates.errors.permission` for example. Do these need to be unique, is that why you are not using `METRICS_UPDATE_ERRORS` for these?

Reply:
I changed them back to use `METRICS_UPDATE_ERRORS` for the name like before. I changed these my mistake.